gravitee-am-service/src/main/java/io/gravitee/am/service/utils/vertx/RequestUtils.java

```diff
@@ -93,13 +93,17 @@ public static MultiMap getCleanedQueryParams(HttpServerRequest httpServerRequest
      */
     public static MultiMap getCleanedQueryParams(String uri) {
         final MultiMap queryParams = getQueryParams(uri);
-        queryParams.remove(ConstantKeys.ERROR_PARAM_KEY);
```

Comment:
why are 96-102 lines removed? If it is by purpose the method's comment is invalid.

Reply:
> why are 96-102 lines removed? If it is by purpose the method's comment is invalid.

I do not get you point. I have extracted a method named cleanParams, behaviour should be the same